RESTserver/src/main/java/com/softserve/if072/restservice/service/HistoryService.java

```diff
@@ -54,10 +54,10 @@ public void update(History history) throws DataNotFoundException {
         LOGGER.info(String.format(successfullyOperation, history.getProduct().getName(), "updated in"));
     }
 
-    public void delete(History history) throws DataNotFoundException {
-        if (historyDAO.delete(history) == 0) {
-            throw new  DataNotFoundException(String.format(historyNotFound, "invalid DELETE operation", history.getProduct().getName()));
+    public void delete(int historyId) throws DataNotFoundException {
```

Comment:
If DataNotFoundException is RuntimeException, then it is not needed to declare in the method signature